### PR TITLE
Update publish process for psd1 signing

### PR DESCRIPTION
**Pull request by VeryEarly**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by isra-fel** [comment]
Like we discussed, as a user, I would very much like to know what these scripts do, and their usage

**Comment by VeryEarly** [comment]
> Like we discussed, as a user, I would very much like to know what these scripts do, and their usage

comments in new scripts added

**Comment by VeryEarly** [comment]
installation and smoke tests passed

File: tools/PackModules.ps1 [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+if (!(Test-Path -Path $tmp)) {
+    throw "tmp folder doesn't exist"
+}
+
+try {
+    foreach ($artifact in Get-ChildItem $tmp -Directory) {
+        $module_name = ($artifact).Name
+        Write-Output "Repackaging $module_name"
+        Update-NugetPackage -TempRepoPath (Get-Item $Artifacts).FullName -ModuleName $module_name -DirPath $tmp"\"$module_name -NugetExe $NugetExe
+        Write-Output "Removing temporary folder $artifact"
```
Comment (isra-fel) [inline tools/PackModules.ps1]:
```suggestion
        Write-Output "Removing $tmp\$module_name"
```

**Pull request merged**